src/file/file.model.ts

```diff
@@ -91,20 +96,20 @@ const handleE11000 = function (error: MongoError, _: any, next: NextFunction) :
  * @return string with the unique fields names and values.
  */
 function getMongoErrorIndex(error: MongoError) : string {
-  // extract the fields names in the MongoError
+  // Extract the fields names in the MongoError
   const fieldsRegex : RegExp = new RegExp(/index\:\ (?:.*\.)?\$?(?:([_a-z0-9]*)(?:_\d*)|([_a-z0-9]*))\s*dup key/i);
   const fieldsMatch : RegExpMatchArray =  error.message.match(fieldsRegex);
   let indexName : string = fieldsMatch[1] || fieldsMatch[2];
 
-  // prettify fields names
+  // Prettify fields names
   indexName = indexName.replace(new RegExp('_1_', 'g'), ', ');
 
-  // extract the fields values of the error thrown
+  // Rxtract the fields values of the error thrown
```

Comment:
please don't change code that isn't related to your issue. it's confusing fr the pr.
If you have comments, you may write them in the related pr when it is in review, 
open an issue (if it is that much of an issue), or leave it as it is and wait for refactor.
Also, typo.

Reply:
?

Reply:
?